search-app/src/cmr/search/api/concepts_lookup.clj

```diff
@@ -119,6 +120,9 @@
       {params :params headers :headers ctx :request-context}
       ;; XXX REMOVE this check and the stubs once the service and
       ;;     the associations work is complete
-      (if (headers "cmr-prototype-umm")
-        (stubs/handle-prototype-request path-w-extension params headers)
+      (if (= "true" (string/lower-case (or (headers "cmr-prototype-umm") "")))
```

Comment:
nit: use the existing `cmr.common.util/safe-lowercase` function instead of `string/lower-case`.

Reply:
Done.